tests/test_binary_sensors.py

```diff
@@ -10,36 +10,64 @@
 
 
 class TestAlertSensor:
+    def test_binary_sensor_is_on(self, hass, config_entry, alarm_device):
+        alarm_device.connect("username", "password")
+        alarm_device.update()
+        coordinator = DataUpdateCoordinator(hass, logging.getLogger(__name__), name="econnect_metronet")
+        entity = AlertSensor("device_tamper", 7, config_entry, "device_tamper", coordinator, alarm_device)
+        assert entity.is_on is True
+
+    def test_binary_sensor_is_off(self, hass, config_entry, alarm_device):
+        alarm_device.connect("username", "password")
+        alarm_device.update()
+        coordinator = DataUpdateCoordinator(hass, logging.getLogger(__name__), name="econnect_metronet")
+        entity = AlertSensor("device_failure", 2, config_entry, "device_failure", coordinator, alarm_device)
+        assert entity.is_on is False
+
+    def test_binary_sensor_missing(self, hass, config_entry, alarm_device):
+        alarm_device.connect("username", "password")
+        alarm_device.update()
+        coordinator = DataUpdateCoordinator(hass, logging.getLogger(__name__), name="econnect_metronet")
+        entity = AlertSensor("test_id", 1000, config_entry, "test_id", coordinator, alarm_device)
+        assert entity.is_on is False
+
+    def test_binary_sensor_anomalies_led_is_off(self, hass, config_entry, alarm_device):
```

Comment:
Do the same test for `anomalies_led_is_on` as we should check all the paths with our test. To do that, simply:
```
alarm_device._inventory[1]["status"] = 2
```

Or something similar.